docs/clients/hobby_input.rst

```diff
@@ -109,10 +109,18 @@ Message Table
 
 Type ID 76 | Hobby Input
 
-+--------+-------------------+----------------+-----------+------+-----------------------------------------------------------------+
-| Sub ID |    Short Name     |     Access     | Data Type | Unit |                              Note                               |
-+========+===================+================+===========+======+=================================================================+
-| 0      | allowed_protocols | get, set, save | uint8     | enum | Standard PWM = 1, OneShot125 = 2, OneShot42 = 3, MultiShot = 4, |
-+--------+-------------------+----------------+-----------+------+-----------------------------------------------------------------+
-| 1      | protocol          | get, set, save | uint8     | enum |                                                                 |
-+--------+-------------------+----------------+-----------+------+-----------------------------------------------------------------+
++--------+--------------------------+----------------+-----------+------+-----------------------------------------------------------------+
+| Sub ID | Short Name               | Access         | Data Type | Unit | Note                                                            |
++========+==========================+================+===========+======+=================================================================+
+| 0      | allowed_protocols        | get, set, save | uint8     | enum | Standard PWM = 1, OneShot125 = 2, OneShot42 = 3, MultiShot = 4, |
++--------+--------------------------+----------------+-----------+------+-----------------------------------------------------------------+
+| 1      | protocol                 | get, set, save | uint8     | enum |                                                                 |
```

Comment:
Description: "The currently active protocol for hobby input. Should not generally need to be set by users, sending hobby input sets this automatically."

Also this can't really be saved, saving it does nothing.

The note here on enums should be the same as allowed_protocols